src/blockencodings.h

```diff
@@ -22,7 +22,8 @@ struct TransactionCompressor {
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
-        READWRITE(tx); //TODO: Compress tx encoding
+        // Future transaction compression standards could be implemented here
```

Comment:
I'd still call it actionable, you just have to bump the cmpctblock protocol version and write a BIP to do so.